wgengine/netstack/netstack.go

```diff
@@ -374,6 +374,12 @@ func (ns *Impl) injectOutbound() {
 }
 
 func (ns *Impl) injectInbound(p *packet.Parsed, t *tstun.Wrapper) filter.Response {
+	if ns.onlySubnets && tsaddr.IsTailscaleIP(p.Dst.IP) {
```

Comment:
IsTailscaleIP is slightly lazy and potentially incorrect down the road if we start handing out non-CGNAT Tailscale IPs to people.

But it's also very easy and cheap compared to comparing netmaps.

I could put some `atomic.Value` of `func(netaddr.IP) bool` in the `netstack.Impl` type I guess.

Reply:
Like we already have in `wgengine/userspace.go`:

```go
        // isLocalAddr reports the whether an IP is assigned to the local                                                                                                               
        // tunnel interface. It's used to reflect local packets                                                                                                                         
        // incorrectly sent to us.                                                                                                                                                      
        isLocalAddr atomic.Value // of func(netaddr.IP)bool                                                                                                                             
```

Reply:
s'fine, we assume IsTailscaleIP in a ton of places already.